lib/segment/src/index/field_index/bool_index/mmap_bool_index.rs

```diff
@@ -138,6 +161,58 @@ impl MmapBoolIndex {
 
         indexed_count
     }
+
+    pub fn get_telemetry_data(&self) -> PayloadIndexTelemetry {
+        PayloadIndexTelemetry {
+            field_name: None,
+            points_count: self.indexed_count as usize,
+            points_values_count: (self.trues_count + self.falses_count) as usize,
```

Comment:
Aren't we double counting the points which have `true` and `false` entries?

Reply:
Wait, the field is named `points_values_count`, I guess it is on purpose :sweat_smile:

Reply:
It is the count of all the values being indexed